src/services/event.ts

```diff
@@ -728,8 +728,8 @@ const toSplit = ['targetPopulations', 'reasons'];
 
 const replacements: Record<string, string> = {
   'Preschool (ages 3-5)': 'Preschool Children (ages 3-5)',
-  'Pregnant Women/Pregnant People': 'Pregnant Women / Pregnant Persons',
-  'Pregnant Women': 'Pregnant Women / Pregnant Persons',
+  'Pregnant Women/Pregnant People': 'Expectant families',
```

Comment:
Do we need to update the property name?

Reply:
the key is the value that will be discarded, so no. If someone tried to import "Pregnant Women/Pregnant People" it would be switched to "Expectant families," which seems like what we want.